docs/source/spawners.md

```diff
@@ -163,4 +163,6 @@ When `Spawner.start` is called, this dictionary is accessible as `self.user_opti
 [Spawner]: https://github.com/jupyter/jupyterhub/blob/master/jupyterhub/spawner.py
 
 
-## Writing a custom spawner
\ No newline at end of file
+## Writing a custom spawner
+
+If you are interested in building a custom spawner, you can read [this tutorial](http://jupyterhub-tutorial.readthedocs.io/en/latest/spawners.html)
```

Comment:
Hi @Scrypy. Thanks for the contribution. One small change would be to add a period at the end of the sentence. Thanks!